src/mme-app/Makefile

```diff
@@ -42,7 +45,11 @@ LIBS :=	-lpthread \
         -lgpr \
         -lprotobuf \
         -lprometheus-cpp-pull -lprometheus-cpp-core -lz \
-	-lsecutil
+	-lsecutil \
+	-lepc \
```

Comment:
In order to avoid stale library installations. Can you please please tell why these libraries are required ?

Reply:
These are library of epctool. It is necessary for DNS code to work.